install/installer/scripts/kots-install.sh

```diff
@@ -30,6 +30,26 @@ catch() {
     exit "${1}"
 }
 
+stop_running_workspaces() {
+    echo "Gitpod: shut down any running workspaces/image-builders"
+
+    # Create a context for gpctl
+    kubectl config set-context "${NAMESPACE}"
+    kubectl config use-context "${NAMESPACE}"
+    # gpctl doesn't have a namespace flag
+    kubectl config set-context --current --namespace="${NAMESPACE}"
+
+    # Get list of workspace instances from gpctl
+    for instance in $(/app/gpctl workspaces list -o json | jq -r 'select(. != null) | .[] | .Instance'); do
+        echo "Gitpod: shutting down workspace ${instance}"
+        /app/gpctl workspaces stop "${instance}"
```

Comment:
This command can be a bit flakey if (presumably) the gRPC server communication errors - I've seen logs like:

```
time="2022-09-26T17:20:18Z" level=fatal msg="cannot connect" error="error upgrading connection: error dialing backend: EOF"
```

Three options as I see it:
1. Do nothing and let it error. A redeploy will almost always be fine
2. Add a `|| true` and never let it error. This allows for missing backups
3. Add a `|| sleep 10 && /app/gpctl workspaces stop "${instance}"` to allow it to retry after 10 seconds

Reply:
Option 3 sounds like the best balance of reliability and safety and should catch most of the spurious gRPC server errors; I'd favor that.

Reply:
Ok, I'll make that change now so you can get the RC out